AK/StringView.cpp

```diff
@@ -36,24 +36,8 @@ StringView::StringView(const ByteBuffer& buffer)
 
 Vector<StringView> StringView::split_view(const char separator, bool keep_empty) const
 {
-    if (is_empty())
-        return {};
-
-    Vector<StringView> v;
-    size_t substart = 0;
-    for (size_t i = 0; i < length(); ++i) {
-        char ch = characters_without_null_termination()[i];
-        if (ch == separator) {
-            size_t sublen = i - substart;
-            if (sublen != 0 || keep_empty)
-                v.append(substring_view(substart, sublen));
-            substart = i + 1;
-        }
-    }
-    size_t taillen = length() - substart;
-    if (taillen != 0 || keep_empty)
-        v.append(substring_view(substart, taillen));
-    return v;
+    StringView seperator_view { &separator, 1 };
```

Comment:
Isn't this pretty inefficient? Splitting by a `StringView` seems a lot more computationally complex than splitting by a `char`.

Reply:
I guess I can write a benchmark to compare before and after.

Reply:
Either way, seperator => separator :^)